lib/auth/auth_with_roles_okta_rbac_test.go

```diff
@@ -421,3 +428,300 @@ func TestOktaMayNotCreateBotUser(t *testing.T) {
 	require.Error(t, err)
 	require.True(t, trace.IsAccessDenied(err), "Expected access denied, got %T: %s", err, err.Error())
 }
+
+func TestOktaServiceLockCRUD(t *testing.T) {
```

Comment:
We should verify non Okta services attempting to create Okta locks as well.

It seems like this would benefit from a table driven approach here. What do you think?

Reply:
I tried that in my first pass. Because each test has a subtly different setup and action to take, I ended up having largely the same code as the subtest approach - just split out into functions included in the table entry. 

In the end I found it easier to read if I just to made each one an independent subtest.

Reply:
> We should verify non Okta services attempting to create Okta locks as well.

Done